core/input.js

```diff
@@ -147,10 +150,14 @@ Blockly.Input.prototype.removeField = function(name) {
         // Removing a field will cause the block to change shape.
         this.sourceBlock_.bumpNeighbours();
       }
-      return;
+      return true;
     }
   }
-  throw Error('Field "%s" not found.', name);
+  if (opt_quiet) {
+    return false;
+  } else {
+    throw Error('Field "%s" not found.', name);
```

Comment:
BTW I have a bug report against this error. The string doesn't format like one would think: https://github.com/google/blockly/issues/3966

Reply:
Thanks for letting me know. Just updated.